### PR TITLE
Summary component [WIP]

### PR DESCRIPTION
**Pull request by holahula**
Finished summary component + page.

Still need to decide how to pass params, and clean up code

### CODE REVIEW

**Comment by holahula** [comment]
What the page currently looks like

Screen Shot 2019-06-07 at 3 20 34 AM

File: src/components/Header.jsx [hunk 1]
```diff
@@ -7,22 +7,38 @@ import PageIndicatorContainer from "../containers/PageIndicatorContainer";
...
+            <img className="logo" src={`/${logo}`} alt="Logo" />
+          </div>
+          <h2 className="text header">Dataset Bias Assessment Tool</h2>
+          <p className="text subheader">
+            Section {currentPage+1}/{totalPages}
+          </p>
+          <h1 className="text title">Progress Title</h1>
+          <div className="indicator-container">
+            <PageIndicatorContainer pageNum={currentPage} />
+          </div>
```
Comment (suzyng83209) [inline src/components/Header.jsx:38]: Oh this Header stuff is something I'll was going to take on. This seems like a good direction though, I can build off of it.

File: src/components/SummaryBox.jsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+          </div>
+        </div>
+        <div id="rectangle" className="green-b"></div>
+      </div>
+    );
+    }
+  }
+}
+
+export default SummaryBox;
```
Comment (suzyng83209) [inline src/components/SummaryBox.jsx:50]: It seems like there's some duplicated html elements in here.
Can you nest the selected child like:
```
<div className="figmaCard">
  <div className="boxboxbox">
    <h1 className="sectionNum"> Section {sectionNumber} </h1>
    <h1 className="sectionTitle"> {sectionTitle} </h1>
  </div>
  <div id="bar" />
  <div className={`outer-circle ${val == 0? 'red-b' : ''}`}>
    <div className="inner-circle">
      <p className={`circle-text ${val == 0? 'red-c' : ''}`}>
    ...
```

Comment (suzyng83209) [reply src/components/SummaryBox.jsx:50]: This is a way of passing in css classNames so your if statements generate the same html elements but with different styles.

**Review by suzyng83209 — CHANGES_REQUESTED** [review]
Some of your Header work will be continued upon with my task. The original task wasn't very clear about the fact that the Header was a separate task but thanks for starting work on it!!

I requested changes on the way that the conditional rendering is handled though. If you take a look at my comment, it should include a useable example.

**Review by suzyng83209 — APPROVED** [review]

**Pull request merged**